assignment/src/main/java/nerdschool/bar/Pub.java

```diff
@@ -1,69 +1,7 @@
 package nerdschool.bar;
 
+// A copy for commenting code
+	
 public class Pub {
 
-
-    public static final String ONE_BEER = "hansa";
-    public static final String ONE_CIDER = "grans";
-    public static final String A_PROPER_CIDER = "strongbow";
-    public static final String GT = "gt";
-    public static final String BACARDI_SPECIAL = "bacardi_special";
-
-    public int computeCost(String drink, boolean student, int amount) {
-
-        if (amount > 2 && (drink == GT || drink == BACARDI_SPECIAL)) {
-            throw new RuntimeException("Too many drinks, max 2.");
-        }
-        int price;
-        if (drink.equals(ONE_BEER)) {
-            price = 74;
-        }
-        else if (drink.equals(ONE_CIDER)) {
-            price = 103;
-        }
-        else if (drink.equals(A_PROPER_CIDER)) price = 110;
-        else if (drink.equals(GT)) {
-            price = ingredient6() + ingredient5() + ingredient4();
-        }
-        else if (drink.equals(BACARDI_SPECIAL)) {
-            price = ingredient6()/2 + ingredient1() + ingredient2() + ingredient3();
```

Comment:
Lines 26 et 29 : too much logic for computeCost method
- This calculation should be done elsewhere
- We need a proper data structure + methods for drinks